### PR TITLE
Fix Sample App

### PR DESCRIPTION
**Pull request by fjnoyp**
Remove cognito feetch auto session example - this needs to be fixed on Amplify iOS first

Clarify key points for using the sample in the Readme.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+Sample flutter app for getting started with the Amplify Flutter Library.  This example uses the Auth, Analytics, and Storage components of the Flutter library. 
 
 Please check out our docs here:
 https://docs.amplify.aws/start/q/integration/flutter
 
-!!! IMPORTANT !!!
+# TL;DR 
+
+**The app will not compile.  You must configure 'amplifyconfiguration.dart' using Amplify CLI.**
+
```
Comment (haverchuck) [inline example/README.md:11]: I don't think we should say this.  The phrase 'The app will not compile' makes it sound like the app  will NEVER compile. Also, they shouldn't have to use the CLI. Customers can (and many do) create their own config files independent of the CLI.

File: example/README.md [hunk 2]
```diff
@@ -1,28 +1,45 @@
...
+**Operations such as logging in or uploading images takes time.  Don't click on the app until they complete.**
+
+
+## Getting Started
+
+**IMPORTANT** 
 
-The app will not compile until you have used the Amplify CLI to configure the
+**The app will not compile until you have used the Amplify CLI to configure** the
 AWS resources necessary for running this app.
```
Comment (haverchuck) [inline example/README.md:20]: Again, we shouldn't suggest they MUST use the CLI (I realize this is what it said before).  Maybe just say '...use the Amplify CLI to configure AWS resources necessary for running this app, or create your own amplifyconfiguration.dart file using the example in our documentation'.

File: example/README.md [hunk 3]
```diff
@@ -1,28 +1,45 @@
...
-https://docs.amplify.aws/lib/project-setup/create-application/q/platform/flutter#n3-provision-the-backend-with-amplify-cli
+https://docs.amplify.aws/start/getting-started/add-api/q/integration/flutter#setup-aws-cloud-resources-with-amplify-cli
+
+Afterwards, run `amplify configure` for this project, then call `add analytics`, `add auth`, and `add storage` before pushing.
 
 Running these steps will generate the `amplifyconfiguration.dart` file within the `lib` folder. 
 
+## Important Notes 
+
+This is a very basic app that interacts with AWS resources.  We did not implement UI showing that the app is "loading" or "uploading" something from AWS.  Some operations like logging in or uploading an image can take some time.  If you do an action and the app doesn't immediately respond, please refrain from clicking away or trying to click a button again - the operation is in progress and interacting with the UI can interfere with that.
```
Comment (haverchuck) [inline example/README.md:38]: I would shorten this to "This is a very basic app that interacts with AWS resources.  We did not implement UI showing that the app is "loading" or "uploading" something from AWS.  Some operations like logging in or uploading an image can take some time. "

File: example/README.md [hunk 4]
```diff
@@ -1,28 +1,45 @@
...
+
+Afterwards, run `amplify configure` for this project, then call `add analytics`, `add auth`, and `add storage` before pushing.
 
 Running these steps will generate the `amplifyconfiguration.dart` file within the `lib` folder. 
 
+## Important Notes 
+
+This is a very basic app that interacts with AWS resources.  We did not implement UI showing that the app is "loading" or "uploading" something from AWS.  Some operations like logging in or uploading an image can take some time.  If you do an action and the app doesn't immediately respond, please refrain from clicking away or trying to click a button again - the operation is in progress and interacting with the UI can interfere with that.    
+
+Please note when signing up that you MUST provide a country code for a new user's phone number.  For example, if you number is American, you will need to append +1 at the beginning.
```
Comment (haverchuck) [inline example/README.md:40]: Good call on this one.

File: example/README.md [hunk 5]
```diff
@@ -1,28 +1,45 @@
...
 Running these steps will generate the `amplifyconfiguration.dart` file within the `lib` folder. 
 
+## Important Notes 
+
+This is a very basic app that interacts with AWS resources.  We did not implement UI showing that the app is "loading" or "uploading" something from AWS.  Some operations like logging in or uploading an image can take some time.  If you do an action and the app doesn't immediately respond, please refrain from clicking away or trying to click a button again - the operation is in progress and interacting with the UI can interfere with that.    
+
+Please note when signing up that you MUST provide a country code for a new user's phone number.  For example, if you number is American, you will need to append +1 at the beginning.  
+
+Lastly, if you want to use this example project as a starting point for your own flutter app, please beware that changing the name of a pre-existing flutter project can be somewhat tricky.  It may be easier to simply copy paste the contents of this example app into a new blank flutter app that you create separately. 
+
```
Comment (haverchuck) [inline example/README.md:43]: I would take this out.

File: example/README.md (hunk 1 above)
Comment (fjnoyp) [reply example/README.md:11]: Alright I will remove this section.

File: example/README.md (hunk 2 above)
Comment (fjnoyp) [reply example/README.md:20]: thanks added your suggestion

File: example/README.md (hunk 3 above)
Comment (fjnoyp) [reply example/README.md:38]: done

File: example/README.md (hunk 4 above)
Comment (fjnoyp) [reply example/README.md:40]: nice!

File: example/README.md (hunk 5 above)
Comment (fjnoyp) [reply example/README.md:43]: done, removed

File: example/README.md [hunk 6]
```diff
@@ -1,28 +1,36 @@
...
 
-Afterwards, run `amplify configure` for this project, then `add analytics`, `auth`, and `storage` before pushing.
-If you haven't used the Amplify CLI before, please follow the link below on using Amplify CLI to configure a flutter project:
+If you haven't used Amplify CLI before, please follow the link below on setting up Amplify CLI to configure a flutter project:
 
 Please see the docs here:
-https://docs.amplify.aws/lib/project-setup/create-application/q/platform/flutter#n3-provision-the-backend-with-amplify-cli
+https://docs.amplify.aws/start/getting-started/add-api/q/integration/flutter#setup-aws-cloud-resources-with-amplify-cli
+
+Afterwards, run `amplify configure` for this project, then call `add analytics`, `add auth`, and `add storage` before pushing.
```
Comment (haverchuck) [inline example/README.md:25]: "amplify configure" should be "amplify init"
